bounded-static/src/lib.rs

```diff
@@ -1706,6 +1709,23 @@ mod smallvec_tests {
         ensure_static(small_vec.to_static());
         ensure_static(small_vec.into_static());
     }
+
+    #[cfg(feature = "alloc")]
+    mod alloc_smallvec_tests {
```

Comment:
nit: it is probably overkill to have a module for this, annotating the test directly is fine since there is only one.

Reply:
Thanks, I'm still not familiar with Rust full ecosystem, so that's nicer if it works.